x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_about_rule/schema.tsx

```diff
@@ -305,6 +305,23 @@ export const schema: FormSchema<AboutStepRule> = {
     ),
     labelAppend: OptionalFieldLabel,
   },
+  setup: {
+    type: FIELD_TYPES.TEXTAREA,
+    label: i18n.translate(
+      'xpack.securitySolution.detectionEngine.createRule.stepAboutRule.setupLabel',
+      {
+        defaultMessage: 'Setup guide',
+      }
+    ),
+    helpText: i18n.translate(
+      'xpack.securitySolution.detectionEngine.createRule.stepAboutRule.setupHelpText',
+      {
+        defaultMessage:
+          'This guide will appear on the rule details page and in timelines (as notes) created from detection alerts generated by this rule.',
```

Comment:
I just put what was in the design mocks in here but clearly we need something better fitted for the setup guide field. Any thoughts? cc @joepeeples @approksiu

Reply:
How about something like:

Provide instructions on rule prerequisites such as required integrations, configuration steps, and anything else needed for the rule to work correctly.